server/views/pages/goal/create/index.njk

```diff
@@ -27,7 +27,7 @@
         <input type="hidden" name="prisonNumber" value="{{ form.prisonNumber }}" />
 
```

Comment:
This is the only place that needed this fix. I've been through all the nunjucks templates looking for where we reference `{{ prisonerSummary.firstName }}` and `{{ prisonerSummary.lastName }}`
This is the only place where we use it as part of setting a nunjucks variable, and is the only screen that has this problem.
All other screens where we reference the prisoner name do so within HTML tags, so I guess the rendering/escaping treatment is different based on whether you use it directly in an element/markup, or in a nunjucks variable.